packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -45,8 +57,72 @@ const SelectedPersonaInner = React.memo(
       getExpandedItems,
       styles,
       theme,
+      dragDropHelper,
+      dragDropEvents,
+      eventsToRegister,
     } = props;
     const itemId = getId();
+    const [root, setRoot] = React.useState<HTMLElement | undefined>();
+    const [dragDropSubscription, setDragDropSubscription] = React.useState<IDisposable>();
+    const [isDropping, setIsDropping] = React.useState(false);
+    const [droppingClassNames, setDroppingClassNames] = React.useState('');
+    const [droppingClassName, setDroppingClassName] = React.useState('');
```

Comment:
does it make sense to put DEFAULT_DROPPING_CSS_CLASS here as default value .. basically React.useState(DEFAULT_DROPPING_CSS_CLASS); ?